lib/state/revision/reducer.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REVISION_IS_VIEWING,
+	REVISION_SELECT,
+} from '../action-types';
+
+export const isViewingRevisions = ( state = false, { type, isViewing } ) =>
+	REVISION_IS_VIEWING === type ? isViewing : state;
+
+export const selectedRevision = ( state = null, { type, revision } ) =>
+	REVISION_SELECT === type ? revision : state;
+
+export default combineReducers( {
+	isViewingRevisions,
```

Comment:
`isViewingRevisions` toggles `<RevisionSelector>`
`selectedRevision` contains the currently selected revision that is displayed in the note content, which can be either a revision or `null` for when the selected revision is the current version of the note.

This mimics the current `<NoteEditor>` [internal state](https://github.com/Automattic/simplenote-electron/blob/master/lib/note-editor.jsx#L46):
```js
{
   revision: null,
   isViewingRevisions: false
}
```

The former could be better renamed. Maybe something like:
`isRevisionSelectorOpen` / `REVISION_SELECTOR_OPEN`